packages/grid/x-grid-modules/src/hooks/utils/useLogger.ts

```diff
@@ -40,31 +55,28 @@ const getAppender = (name: string, logLevel: string, appender: Logger = console)
   }, {} as any);
 
   return logger as Logger;
-};
+}
 
 const defaultFactory: (logLevel: string) => LoggerFactoryFn = (logLevel: string) => (
   name: string,
-) => {
-  if (!isDebugging) {
-    return noopLogger;
-  }
-  return getAppender(name, logLevel);
-};
+) => getAppender(name, logLevel);
 
-export type LoggerFactoryFn = (name: string) => Logger;
+type LoggerFactoryFn = (name: string) => Logger;
 
 // TODO Refactor to allow different logger for each grid in a page...
 let factory: LoggerFactoryFn | null;
+
 export function useLoggerFactory(
   customLogger?: Logger | LoggerFactoryFn,
-  logLevel: string | boolean = 'debug',
+  logLevel: string = 'debug',
```

Comment:
the boolean was to turn off logging

Reply:
You are right. I have restored it in a follow up pull request. If you check master it should be back in here. It was creating TypeScript errors in the demos/stories but wasn't catch by the CI.

The CI part is now covered by building in Netlify the current commit, it's not ideal but it's a good first safeline.